### PR TITLE
Add Awesome COBOL

### PR DESCRIPTION
**Pull request by augustopedro**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 - [AutoIt](https://github.com/J2TeaM/awesome-AutoIt)
 - [Crystal](https://github.com/veelenga/awesome-crystal)
 - [TypeScript](https://github.com/dzharii/awesome-typescript)
+- [Cobol](https://github.com/dshimy/awesome-cobol)
```
Comment (sindresorhus) [inline readme.md]: `Cobol` → `COBOL`

**Comment by sindresorhus** [comment]
Hah, fun stuff. Though, it haven't been updated in over a year and author doesn't seem to be active on GitHub, so not sure about this. Also missing a license.

**Comment by augustopedro** [comment]
Hahah, indeed.

**Comment by jyguyomarch** [comment]
There is no such thing as "awesome COBOL" :stuck_out_tongue_winking_eye:

**Comment by sindresorhus** [comment]
@augustopedro Would you be interested in forking it and maintaining the fork? Or if not, know anyone that would? Not super happy about adding a list people can't contribute to.

**Comment by augustopedro** [comment]
At the moment, no. If I find someone interested in maintain the fork, I'll let you know. But thanks anyway :+1:

**Comment by robbyoconnor** [comment]
Squash please :)

**Comment by sindresorhus** [comment]
Alright. Merging for now. Opened an issue about finding a maintainer: https://github.com/sindresorhus/awesome/issues/414

**Pull request closed without merging**